src/planscape/rscripts/forsys.R

```diff
@@ -548,7 +549,8 @@ call_forsys <- function(
     return_outputs = TRUE,
     write_outputs = TRUE,
     overwrite_output = TRUE,
-    scenario_name = scenario$name,
+    # scenario_name = scenario$name,
+    scenario_name = scenario$uuid, # using UUID here instead of name
```

Comment:
This does what we want, apparently -- outputting a scenario name that's a full UUID. 
However, it also plugs that scenario name into the resulting .json file, which may not be a good UX.

If this isn't okay, I'm not sure of our options. I don't see an alternative arg to [forsys::run] (https://github.com/forsys-sp/forsysr/blob/main/R/forsys_run.R), so unless we were somehow able to add an optional parameter, or otherwise ensure that forsys avoids name collisions, maybe there's something we could do on our side to put each zip inside of a uniquely directory on the server.